src/pre_commit_terraform/terraform_docs_replace.py

```diff
@@ -53,12 +53,13 @@ def invoke_cli_app(parsed_cli_args: Namespace) -> ReturnCodeType:
         'https://github.com/antonbabenko/pre-commit-terraform/issues/248'
         '#issuecomment-1290829226',
         category=UserWarning,
+        stacklevel=2,
```

Comment:
That's breaks test. And because it's deprecated hook, no sense to fix it later, so just set it to 1 and forget about it
```suggestion
        stacklevel=1,
```